### ISSUE TITLE
Philio PST02-A-BR: config parameters are terrible

### ISSUE THREAD

**Issue by AlCalzone** [zwave-js/zwave-js#2301]
including several wrong option definitions that should be partials.

`packages\config\config\devices\0x013c\pst02-a-br.json`
Manual here: https://www.easyfikssmarthome.nl/download/7140093-0-pst02-manual-20150916pdf.pdf

**Comment by orandasoft** [comment]
After the changes I made, is this still an issue?

**Comment by AlCalzone** [comment]
I don't think you touched the [PST02-A-BR file](https://devices.zwave-js.io/?jumpTo=0x013c:0x0002:0x0059:0.0), only the other 3. Maybe this one can be merged into [PST02-A](https://devices.zwave-js.io/?jumpTo=0x013c:0x0002:0x000c:0.0), but I didn't check thoroughly.

**Comment by orandasoft** [comment]
Understood. I didn't realize there were other files besides the three I touched otherwise I would have had a look at it. Shall I take a look?

**Comment by AlCalzone** [comment]
Go for it 👍🏻
